commands/command.go

```diff
@@ -24,6 +24,8 @@ import (
 	"github.com/spf13/cobra"
 )
 
+var errOperationAborted = fmt.Errorf("Operation aborted.")
```

Comment:
Tiny nitpick: This might be more discover-able in `commands/errors.go`

Reply:
I pushed a commit moving this: https://github.com/digitalocean/doctl/pull/1054/commits/b2a1e76d541d47afb3f024281a2cf5721b513b29